### PR TITLE
Use CTest script in continous integration

### PR DESCRIPTION
**Pull request by dalg24**

### CODE REVIEW

**Comment by dalg24** [comment]
TODO:
- [ ] add ``valgrind`` (to be able to set up memcheck)
- [ ] set time zone (needed for CTest timing info)

**Comment by dalg24** [comment]
hey

**Comment by dalg24** [comment]
Ready for review

File: docker/build_cap.sh [hunk 1]
```diff
@@ -1,31 +1,17 @@
...
 su jovyan <<EOF
-export LD_LIBRARY_PATH=${BOOST_DIR}/lib:$LD_LIBRARY_PATH && \
-export PATH=${PYTHON_DIR}/bin:${PATH} && \
-ctest -j2 -V && \
-make coverage-cpp && make coverage-python && \
-sed -i.fixpath "s|/dummy/cap||g" lcov.info && \
+set -x
+export LD_LIBRARY_PATH=${BOOST_DIR}/lib:$LD_LIBRARY_PATH
+export PATH=${PYTHON_DIR}/bin:${PATH}
+ctest -j2 -V -S ${PREFIX}/source/cap/docker/TravisCI.cmake || echo "oh nooo"
```
Comment (Rombur) [inline docker/build_cap.sh]: No need for the echo.

File: docker/build_cap.sh [hunk 2]
```diff
@@ -1,31 +1,17 @@
...
+cd ${PREFIX}/build/cap || echo "here"
```
Comment (Rombur) [inline docker/build_cap.sh]: Same thing.

File: docker/get_build_info.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
+    echo "TODO"
```
Comment (Rombur) [inline docker/get_build_info.sh:4]: What's left to do?

**Comment by Rombur** [comment]
Fix the echo and merge yourself.

File: docker/get_build_info.sh (hunk 3 above)
Comment (dalg24) [inline docker/get_build_info.sh:4]: Write commit message

**Pull request merged**